### PR TITLE
feat(onboarding): initial version of mobile onboarding email

### PR DESCRIPTION
**Pull request by scefali**
This PR adds a new route (`/api/0/organizations/${slug}/onboarding-continuation-email/`) that will send an email for folks to finish onboarding. Note that we are only planning on utilizing the email with regards to having set up onboarding on a mobile device:

Screen Shot 2022-04-25 at 3 17 16 PM

### CODE REVIEW

**Review by PixelDust22 — APPROVED** [review]
LGTM

File: src/sentry/api/endpoints/organization_onboarding_continuation_email.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    platforms = serializers.ListField(
+        child=serializers.CharField(max_length=255),
+    )
+
+
+def get_request_builder_args(user: User, organization: Organization, platforms: List[str]):
+    num_platforms = len(platforms)
+    context = {
+        "recipient_name": user.get_display_name(),
+        "onboarding_link": f"/onboarding/{organization.slug}/?referrer=onboarding_continuation-email",
```
Comment (markstory) [inline src/sentry/api/endpoints/organization_onboarding_continuation_email.py:24]: Why a mix of `_` and `-`?

File: src/sentry/api/endpoints/organization_onboarding_continuation_email.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        "type": "organization.onboarding-continuation-email",
+        "context": context,
+        "template": "sentry/emails/onboarding-continuation.txt",
+        "html_template": "sentry/emails/onboarding-continuation.html",
+    }
+
+
+class OrganizationOnboardingContinuationEmail(OrganizationEndpoint):
+    private = True
+    permission_classes = (OrganizationPermission,)
```
Comment (markstory) [inline src/sentry/api/endpoints/organization_onboarding_continuation_email.py:40]: Do members need to use this endpoint? The `OrganizationPermission` requires `org:write` to make POST requests.

Comment (scefali) [reply src/sentry/api/endpoints/organization_onboarding_continuation_email.py:40]: @markstory This endpoint is specifically geared for owners who do the initial onboarding but it guess wouldn't hurt to open it up to everyone.

**Pull request merged**